pinpayments/views.py

```diff
@@ -0,0 +1,53 @@
+from django.http import HttpResponse
```

Comment:
I have no issue including these examples in the `docs` or `tests` folder,, but they shouldn't be in `pinpayments.views as that should only include views that `django-pinpayments` *provides*, not suggested samples.